Modules/System Tests/Record Link Management/src/RecordLinkMgtTest.Codeunit.al

```diff
@@ -85,10 +85,7 @@ codeunit 132508 "Record Link Mgt. Test"
 
         // [GIVEN] A new record is created to set record links on
         FromRecordLinkRecordTest.DeleteAll();
-        FromRecordLinkRecordTest.Init();
-        FromRecordLinkRecordTest.PK := 1;
-        FromRecordLinkRecordTest.Field := 'Rec A';
-        FromRecordLinkRecordTest.Insert();
+        CreateRecordLinkRecTest(FromRecordLinkRecordTest, 1, 'Rec A');
```

Comment:
I dislike passing constants to helper methods in test code. While this case seems benign, it leads to paremeterized tests (one-liners) where it because very hard to know what is actually being tested. 
In cases like this, what I do is create a well named additional helper in which the constants are passed to the 'general' helper. Something like:
CreateFirstRecordLink(FromRecordLinkRecordTest);

and the second hardcode one would then obviously be:
CreateSecondRecordLink(FromRecordLinkRecordTest);

Another alternative to this is to create variables like RecordLinkPrimaryKeyValue and RecordLinkFieldFieldValue, assign values to those and pass the variables but that would kind of defeat the purpose of your helper here.